rulesets/src/request-response.ruleset.yml

```diff
@@ -68,7 +76,80 @@ rules:
       functionOptions:
         notMatch: "^(x|X)-"
 
+  sps-invalid-location-header:
+    description: Location header must not be present in non-201 responses
+    severity: error
+    given: $.paths[*][*].responses[?(@property !== '201')].headers
+    then:
+      field: Location
+      function: falsy
+
+  sps-authorization-missing:
+    description: Security field MUST be present with at least one item
+    severity: error
+    given: "$"
+    then:
+      field: "security"
+      function: schema
+      functionOptions:
+        schema:
+          type: array
+          minItems: 1
```

Comment:
does this enforce that the security field is defined? or just if it is defined that it has 1?

Reply:
It enforces that its defined with at least 1 item in the array.